--- project/tmp/src/logic/95.agilogic	2025-11-28 17:00:29.507930008 +0000
+++ translations/sq2/logic/95.agilogic	2025-12-03 21:34:39.539111004 +0000
@@ -21,8 +21,7 @@
 
   if (f68) {
     v3 = 255;
-    add.to.pic(26, 0, 0, 66, 62, 0, 4);
-    add.to.pic(26, 0, 1, 66, 88, 0, 4);
+    add.to.pic(26, 0, 0, 60, 62, 0, 4);
   }
   show.pic();
 
@@ -31,20 +30,20 @@
     sound(60, f61);
   } else {
     v67 = 2;
-  
+
     if (v116 == 1) {
       print(m5);
       print(m6);
     }
-  
+
     if (v116 == 2) {
       print(m4);
     }
-  
+
     if (v116 == 3) {
       print(m7);
     }
-  
+
     if (v1 == 46) {
       print(m8);
     }
